TRIAGING.md

```diff
@@ -95,7 +95,7 @@ You can mention him in the relevant thread like this: `@btford`.
 
 > Thanks for submitting this issue!
 > Unfortunately, we don't think this functionality belongs in core.
-> The good news is that you could easily implement this as a third-party module and publish it on Bower and/or to the npm repository.
+> The good news is that you could easily implement this as a third-party module and publish it to the npm registry.
```

Comment:
Do we need this change? It is about other people publishing to bower. Not us using dependencies from bower. Or are you just trying to move everyone away from bower in every possible way?

Reply:
Yeah, I guess I should revert this change.

Reply:
I think it's okay to make this change. bower has no immediate advantages over npm, and is getting less important every week.